stdpopsim/catalog/MusMus/species.py

```diff
@@ -151,7 +151,8 @@
 # size is generally accepted. Here we assign a value of 500000 for the Ne as described
 # in Booker and Keightley (2018) which is probably most consistent with M. m. castaneus.
 # Users of stdpopsim may wish to modify this value or implement a specific demographic
-# model if they wish to simulate data for a different M. musculus subspecies.
```

Comment:
I added this mention of Fujiwara et al. (2022). Otherwise, we're not using it to set any of the current species parameters. I also re-ordered the citation objects below to reflect their order in the documentation above

Reply:
This sounds fine.